docs/development/translations.rst

```diff
@@ -0,0 +1,396 @@
+
+If you find that a *source string* contains terms from the
+:doc:`SecureDrop glossary <../glossary>` or the `EFF Surveillance
+Self-Defense glossary`_, but the glossary sidebar says ``No related
+strings found in the glossary.``, we'd really appreciate it if you
+could add those terms to the glossary of the language you're working
+with.
+
+Glossaries
+----------
+
+Language glossaries on Weblate
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Weblate contains an internal glossary for each language, to which we can add suggested translations. Learn more about :ref:`using language glossaries on Weblate <how_to_use_language_glossaries_on_weblate>`.
+
+If a term is missing from the glossary for the language you're translating into, you can refer to the :ref:`Technical glossaries <technical_glossaries>` for additional context. Then you can contribute to improving your language glossary on Weblate by adding a suggested translation yourself!
+
+.. _technical_glossaries:
+
+Technical glossaries
+^^^^^^^^^^^^^^^^^^^^
+
+Translating SecureDrop requires understanding some complicated security terminology, involving cryptography, networking, and threat models. If you are unfamiliar with terms you encounter in :ref:`source strings <weblate_glossary_source_string>`, consult one of the following glossaries, or :ref:`ask for help <getting_help_translations>`!
+
+- The :doc:`SecureDrop glossary <../glossary>` explains terms specific
+  to SecureDrop.
+- The `EFF Surveillance Self-Defense glossary`_ explains many general
+  security concepts.
+
+Weblate glossary
+^^^^^^^^^^^^^^^^
+
+Here is a list of terms that are specific to the usage of Weblate for SecureDrop.
+
+.. _weblate_glossary_reviewer:
+
+Reviewer
+""""""""
+
+Reviewers are people who are trusted to review and accept new translations into SecureDrop.
+
+.. _weblate_glossary_source_string:
+
+Source string
+"""""""""""""
+
+On Weblate, the phrases being translated are called *source string*. (No relation with the terms *source* and *journalist* in SecureDrop.)
+
+Source strings are English phrases and are automatically extracted from SecureDrop's code. Because of that, they can only be modified by developers outside Weblate. Learn :ref:`how to suggest changes to a source string <how_to_suggest_changes_to_a_source_string>`.
+
+.. _`Weblate`: https://weblate.org/
+.. _`SecureDrop Weblate instance`: https://weblate.securedrop.org/
+.. _`Weblate registration page`: https://weblate.securedrop.org/accounts/register/
+.. _`Weblate dashboard`: https://weblate.securedrop.org/
+.. _`translation category of the SecureDrop forum`: https://forum.securedrop.org/c/translations
+.. _`SecureDrop instant messaging channel`: https://gitter.im/freedomofpress/securedrop
+.. _`Weblate documentation`: https://docs.weblate.org/
+.. _`EFF Surveillance Self-Defense glossary`: https://ssd.eff.org/en/glossary/
+
+
+.. |Weblate registration page screenshot| image::
+   ../images/weblate/registration.png
+   :width: 100%
```

Comment:
Using a `figure` would allow add a caption to let people know that they can: "Click on the image to see it full-size." or something in those lines. My first attempt didn't behave as expected, but I'll investigate.

If there is such caption, I think it would be reasonable to reduce the size of the images / figures so that they don't interrupt the reading flow as much.

Reply:
By the way @eloquence, is it expected for the images not to be displayed when serving the docs locally (`make docs`) or is something wrong with my setup? As far as I can tell, the images are there, but Firefox refuses to display them. I haven't looked too much into this, it may have to do with the `make docs` target. If it's not expected I'll investigate properly.

Reply:
@gonzalo-bulnes Have you installed `git-lfs` (see the setup guide here: https://docs.securedrop.org/en/stable/development/documentation_guidelines.html#updating-documentation)? If not, that would explain why you're seeing broken images, as the files are otherwise just textual pointers to resources that have not yet been downloaded.

Reply:
I didn't install `git-lfs` at first, and I completely forgot about it. I'll install it. Thank you for the answer and explanation!